pandas/tests/io/pytables/test_store.py

```diff
@@ -1009,6 +1010,7 @@ def test_to_hdf_with_object_column_names(tmp_path, setup_path):
             assert len(result)
 
 
+@pytest.mark.skipif(_using_copy_on_write(), reason="strides buggy with cow")
 def test_hdfstore_strides(setup_path):
```

Comment:
Is this caused by the concat changes?

Reply:
Yeah strides changed from 8 to 16, this is really weird

Reply:
Yeah this seems to fix it